Bundle/BusinessEntityBundle/Annotation/AnnotationDriver.php

```diff
@@ -88,7 +88,6 @@ public function getAllClassNames()
             );
             foreach ($iterator as $file) {
                 $sourceFile = realpath($file[0]);
-                require_once $sourceFile;
```

Comment:
Why remove this ? It was introduced in here: https://github.com/Victoire/victoire/commit/12739135d44cda681766c576c0a43989ed8cc212